plugin/assets/css/src/wizard/content.css

```diff
@@ -41,4 +41,9 @@
 	& .material-wizard__card img {
 		width: 100%;
 	}
+
+	& .mdc-button--offsite .mdc-button__icon {
```

Comment:
is this valid css?

Reply:
@rodydavis Yes the plugin turns it into `.material-wizard__content .mdc-button--offsite .mdc-button__icon` in the compiled file